src/symlink.c

```diff
@@ -23,7 +23,9 @@
 #include <unistd.h>
 #endif
 
+#if !defined(MAIN_NAME)
```

Comment:
I'd prefer that we consistently use `#ifndef` in such cases.

Reply:
The file already has an `#if !defined (_MSC_VER)` a few line above this one. So, I consistently use `!defined`.

But, as a side note, the compiler on Windows complains about the `!defined`. I can't see a reason. It works, but gcc complains. So, I'll use `#ifndef `.